src/app/api/api_v1/endpoints/robot_type.py

```diff
@@ -41,7 +42,7 @@ async def read_robot_type(
 async def create_robot_type(
         *,
         db: AsyncSession = Depends(deps.get_async_db),
-        robot_type_in: RobotTypeCreate):
+        robot_type_in: RobotTypeUpdate):
```

Comment:
Leftover from a test? This is not failsafe, as name and vendor needs to be provided and in the update schema both fields are marked as optional. We would lose `pydantic` validations here.